internal/versions/versions.go

```diff
@@ -85,11 +101,11 @@ const (
 
 	// currently supported versions.
 	//nolint:revive
-	V1_24 ValidK8sVersion = "1.24"
+	V1_24 ValidK8sVersion = "1.24.9"
 	//nolint:revive
-	V1_25 ValidK8sVersion = "1.25"
+	V1_25 ValidK8sVersion = "v1.25.6"
 	//nolint:revive
-	V1_26 ValidK8sVersion = "1.26"
+	V1_26 ValidK8sVersion = "v1.26.1"
```

Comment:
Same here, I think we could/will run into some problems if we just start prefixing the version with `v`